extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview.directive.ts

```diff
@@ -82,6 +90,14 @@ angular.module('oppia').directive('oppiaNoninteractiveSkillreview', [
               }
             });
           };
+          // Function to check whether the RTE modal or the concept card
+          // modal should be triggered.
+          ctrl.toggleModal = function(event) {
+            return (
+              event.currentTarget.offsetParent.dataset.ckeWidgetId ||
+              $scope.CkEditorCopyContentService.copyModeActive
```

Comment:
Have you tested these changes? Can you please update the "Proof the changes are correct" section in the PR description?

Reply:
Sure,but what do I add? I can add another updated recording if you like - would that do?

Reply:
The same screen-recording with the new changes! (Asking this as we have updated the PR with the new changes so wanted to ensure that the changes are still correct! :))

Reply:
@DubeySandeep I have uploaded the new recording.

Reply:
Thanks!

I was not expecting `$scope.CkEditorCopyContentService` to be defined, can you please add a log and check?

Reply:
You were right, there was an error. I made a few changes - now it's working perfectly.